haystack/document_stores/memory.py

```diff
@@ -953,14 +954,15 @@ def query(
         index: Optional[str] = None,
         headers: Optional[Dict[str, str]] = None,
         all_terms_must_match: bool = False,
-        scale_score: bool = False,
+        scale_score: bool = True,
```

Comment:
@ZanSara This is the breaking change. Don't you think?

Reply:
Not really, the default should be True and it was False only because it was not supported

Reply:
Ah.
I aligned the parameters of this document store to the default values ​​for other similar document stores...

For example:
https://github.com/deepset-ai/haystack/blob/e8f9b1b65dc8a7f36e6e9bdcd75b177ff4019edd/haystack/document_stores/search_engine.py#L749